rhizome/host/lib/vm_setup.rb

```diff
@@ -345,9 +345,20 @@ def storage(storage_params, storage_secrets, prep)
 
   def download_boot_image(boot_image, custom_url: nil)
     urls = {
-      "ubuntu-jammy" => "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img",
-      "almalinux-9.1" => "https://repo.almalinux.org/almalinux/9/cloud/x86_64/images/AlmaLinux-9-GenericCloud-latest.x86_64.qcow2",
-      "opensuse-leap-15.4" => "https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2",
+      "ubuntu-jammy" => if Arch.arm64?
+                          "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-arm64.img"
+                        elsif Arch.x64?
+                          "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img"
+                        else
+                          fail "BUG: inexhaustive architecture check"
+                        end,
+      "almalinux-9.1" => if Arch.x64?
+                           "https://repo.almalinux.org/almalinux/9/cloud/x86_64/images/AlmaLinux-9-GenericCloud-latest.x86_64.qcow2"
+                         elsif Arch.arm64?
+                           "https://repo.almalinux.org/almalinux/9/cloud/aarch64/images/AlmaLinux-9-GenericCloud-latest.aarch64.qcow2"
+                         else
+                           fail "BUG: inexhaustive architecture check"
+                         end,
```

Comment:
Ubuntu checks `Arch.arm64?` before `Arch.x64?`, whereas Alma checks `Arch.x64?` before `Arch.arm64?`. Maintaining a consistent order could be beneficial.

Reply:
I'll do that